did/plugins/gitlab.py

```diff
@@ -176,9 +177,18 @@ def iid(self):
 
     def __str__(self):
         """ String representation """
-        return "{0}#{1} - {2}".format(
-            self.project['path_with_namespace'],
-            str(self.id).zfill(PADDING), self.title)
+        if self.parent.options.format == "md":
+            return "[{1}#{3}]({0}/{1}/-/{2}/{3}) - {4}".format(
+                self.gitlabapi.url,
+                self.project['path_with_namespace'],
+                ("issues"
+                    if self.data['target_type'] == 'Issue'
```

Comment:
I tried this with my account and for this issue:

* https://gitlab.com/testing-farm/general/-/issues/41

I got a link to a (non-existent) merge request instead:

* https://gitlab.com/testing-farm/general/-/merge_requests/41

Quickly checked the `self.data` and I see this:

    'noteable_type': 'Issue'
    'target_type': 'Note'

Perhaps, we should use `noteable_type` here instead?

Reply:
Sorry, missed a case, fixing.